app/src/ui/dialog/dialog.tsx

```diff
@@ -180,6 +192,12 @@ export class Dialog extends React.Component<IDialogProps, IDialogState> {
       return
     }
 
+    if (this.props.maxWidth !== undefined) {
+      const widthSansPadding = window.innerWidth - 40
```

Comment:
Is that 40 specified/used somewhere else? Just making sure we reuse the constant somehow instead of hardcoding it multiple times.

Reply:
Made a constant